### ISSUE TITLE
[BUG] Invalid link to rule description

### ISSUE THREAD

**Issue by fartem** [dart-code-checker/dart-code-metrics#336]
* **Dart code metrics version:** 3.2.2.
* **Dart sdk version:** 2.12.2.

I'm running plugin from console and link to `avoid-returning-widgets` is invalid.

Console output:

```shell
Warning Avoid returning widgets from a function. : avoid-returning-widgets https://github.com/dart-code-checker/dart-code-metrics/tree/master/doc/rules/avoid-returning-widgets.html
```

It reproduces by running plugin from console from a project that contains issue by `avoid-returning-widgets`.

**Comment by dkrutskikh** [comment]
@fartem looks like https://github.com/dart-code-checker/dart-code-metrics/issues/328#issue-887377787
